cfgov/unprocessed/js/modules/util/assign.js

```diff
@@ -28,11 +28,12 @@ function _isPlainObject( object ) {
 */
 function assign( destination ) {
   destination = destination || {};
-
+  var hasOwnProp;
   for ( var i = 1; i < arguments.length; i++ ) {
     var source = arguments[i] || {};
+    hasOwnProp = Object.hasOwnProperty.bind( source );
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/1866/commits/ce557d2aa816beb344b852015146b1c570d10880